### DISCUSSION TITLE
Searching large timsTOF datasets

### DISCUSSION THREAD

**Discussion by MHBI2018** [vdemichev/DiaNN#842] [Q&A]
Hello,

I currently run DIA-NN on Linux with ~256 GB of ram. I have 100 dia-PASEF files each is about 10 GB in size. It looks like 256 GB is not sufficient so I have to run my searches in portions. Any recommendations on how to do this? I have an extensive spectral library so 2nd passing is not as important and can search against that, but how should I combine my datasets?

Another question, I will be increasing my ram access soon, is there an estimate to how much ram is needed per file size?

Thank you

**Comment by vdemichev** [comment]
256Gb is enough for any DIA-NN search. Please make sure you use the latest DIA-NN 1.8.1
